### PR TITLE
Fix: Map data problems

### PR DESCRIPTION
**Pull request by Faustrox**
**Description:** Fixing problems with map data, hospital data didn't show up.

### CODE REVIEW

**Review by adnanjt — APPROVED** [review]
LGTM

**Review by inteljudajocu — APPROVED** [review]
LGTM

File: app/views/DR/Maps/index.js [hunk 1]
```diff
@@ -18,11 +18,17 @@ const index = () => {
         }}
         swipeEnabled={false}>
         <Tab.Screen
-          name={languages.t('navigation.hospitals_maps')}
+          name={'Hospitals'}
```
Comment (iocedano) [inline app/views/DR/Maps/index.js]:
```suggestion
          name="Hospitals"
```

File: app/views/DR/Maps/index.js [hunk 2]
```diff
@@ -18,11 +18,17 @@ const index = () => {
...
+          options={{
+            tabBarLabel: languages.t('navigation.hospitals_maps'),
+          }}
           component={Map}
         />
         <Tab.Screen
-          name={languages.t('navigation.laboratories_maps')}
+          name={'Laboratories'}
```
Comment (iocedano) [inline app/views/DR/Maps/index.js]:
```suggestion
          name="Laboratories"
```

File: app/views/DR/Maps/index.js (hunk 1 above)
Comment (Faustrox) [reply app/views/DR/Maps/index.js:21]: Why not in plural?

Comment (Faustrox) [reply app/views/DR/Maps/index.js]: Why not in plural? And why in double quote?

Comment (inteljudajocu) [reply app/views/DR/Maps/index.js]: It's still in plural, the suggestion is mostly about the brackets "{ }"

**Pull request merged**